app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -46,22 +52,19 @@
             android:contentDescription="@string/current_profile_picture_content_description"
             android:focusable="true"
             app:add_shadow="true"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
             app:shadow_radius="8"
             app:srcCompat="@drawable/ic_default_avatar" />
 
           <ImageView
             android:id="@+id/add_profile_activity_edit_user_image_view"
-            android:layout_width="48dp"
-            android:layout_height="48dp"
+            android:layout_width="41dp"
+            android:layout_height="41dp"
```

Comment:
Two reasons why this is incorrect:
1. Sizes should be in multiple of 4.
2. Any clickable item should have atleast 48x48dp size otherwise it will fail accessibility tests.

Also did you get chance to look at #4139 or #4155 ? This might help in this.